docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
-| MB_SOURCE_ADDRESS_HEADER                   | string                                   | `X-Forwarded-For`                                              | Identify the source of HTTP requests by this headers value, instead of its remote address.                                                                                                                                                                                                                 |
-| MB_SSL_CERTIFICATE_PUBLIC_KEY              | string                                   | `null`                                                         | Base-64 encoded public key for this sites SSL certificate.Specify this to enable HTTP Public Key Pinning.See http://mzl.la/1EnfqBf for more information.                                                                                                                                                   |
+Many settings in Metabase can viewed and modified in the Admin Panel, or set via environment variables. The environment variables always take precedence.
+
+Setting environment variables can be done in various ways depending on how Metabase is being run.
+
+JAR (Note that on Windows, use `set` instead of `export`):
+
+```
+export MB_SITE_NAME="Awesome Company"
+java -jar metabase.jar
+```
+
+JAR alternative (declared with `-D` prepending the variable):
+
+```
+java -DMB_SITE_NAME="Awesome Company" -jar metabase.jar
+```
+
+Docker:
+
+```
+docker run -d -p 3000:3000 -e 'MB_SITE_NAME=Awesome Company' --name metabase metabase/metabase
+```
+
+---
+
+#### `MAX_SESSION_AGE`
+
+Type: integer<br>
+Default: `20160`
+
+Session expiration, defined in seconds (default is 2 weeks), which will logout users after the defined period and require re-authentication.
+
+Note: This setting is not an idle/inactivity timeout. If you set this to 15 minutes, your users have to login (or re-authenticate) again every 15 minutes.
+
+Use [MB_SESSION_COOKIES](#MB_SESSION_COOKIES) to only expire sessions, when browser is closed.
+
+#### `MB_ADMIN_EMAIL`
+
+Type: string<br>
+Default: `null`
+
+The email address users should be referred to if they encounter a problem.
+
+#### `MB_ANON_TRACKING_ENABLED`
+
+Type: boolean<br>
+Default: `true`
+
+Enable the collection of anonymous usage data in order to help Metabase improve.
+
+#### `MB_API_KEY`
+
+Type: string<br>
+Default: `null`
+
+Middleware that enforces validation of the client via the request header `X-Metabase-Apikey`. If the header is available, then it’s validated against `MB_API_KEY`. When it matches, the request continues otherwise it’s blocked with a 403 Forbidden response.
+
+#### `MB_APPLICATION_COLORS`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"{}"`
+
+JSON object of primary colors used in charts and throughout Metabase.
```

Comment:
Does this need an example for how to structure the JSON object?

Reply:
It would be great to have more extensive examples of most of the variables, but that's wasn't the focus of this PR.
There's a handful of this type of variable, and they're all rather large/complicated - hence it's JSON and not integer/etc.